lisa/tools/hyperv.py

```diff
@@ -206,6 +208,20 @@ def get_default_external_switch(self) -> Optional[VMSwitch]:
             'Get-VMSwitch | Where-Object {$_.SwitchType -eq "External"} '
             "| Select -First 1 | select Name | ConvertTo-Json",
             force_run=True,
+            fail_on_error=False,
+        )
+
+        if not switch_json:
+            return None
+
+        return VMSwitch.from_json(switch_json)  # type: ignore
+
+    def get_default_internal_switch(self) -> Optional[VMSwitch]:
```

Comment:
It doesn't need to create another method with similar functionality. Add an argument to set returned types.

Reply:
Fixed